common/constants/default.go

```diff
@@ -17,7 +16,9 @@ const (
 	NodeName = "NodeName"
 
 	ProjectName = "KubeEdge"
-	SystemName  = "kubeedge"
+	SystemName  = "kubeedge-system"
```

Comment:
modified to kubeedge-system

Reply:
will this cause version compatibility issue?

Reply:
On the cloud, we init this namespace field by executing NamespaceInit function.  On the edge, if the namespace kubeedge cannot get, will try the namespace kubeedge-system.

Reply:
this is OK, but there're some deployment YAMLs in dir https://github.com/kubeedge/kubeedge/tree/master/build/cloud still using ns `kubeedge`, do we also need to fix those synchronously?

Reply:
or maybe we can just focus on helm install.

Reply:
> this is OK, but there're some deployment YAMLs in dir https://github.com/kubeedge/kubeedge/tree/master/build/cloud still using ns kubeedge, do we also need to fix those synchronously?

I will fix it.

Reply:
Do the labels and the namespace all needed to modify?
```
apiVersion: v1
kind: ServiceAccount
metadata:
  labels:
    k8s-app: kubeedge  => k8s-app: kubeedge-system
    kubeedge: cloudcore =>  kubeedge-system:  cloudcore
  name: cloudcore
  namespace: kubeedge  => kubeedge-system
```

Reply:
IMHO, we just need to modify the namespace. 👀